examples/01_standalone_sdk/25_agent_delegation.py

```diff
@@ -0,0 +1,88 @@
+"""
+Agent Delegation Example
+
+This example demonstrates the agent delegation feature where a main agent
+delegates tasks to sub-agents for parallel processing.
+Each sub-agent runs independently and returns its results to the main agent,
+which then merges both analyses into a single consolidated report.
+"""
+
+import os
+import time
+
+from pydantic import SecretStr
+
+from openhands.sdk import (
+    LLM,
+    Conversation,
+    get_logger,
+)
+from openhands.tools.preset.default import get_default_agent
+
+
+logger = get_logger(__name__)
+
+# Configure LLM and agent
+# You can get an API key from https://app.all-hands.dev/settings/api-keys
+api_key = os.getenv("LLM_API_KEY")
+assert api_key is not None, "LLM_API_KEY environment variable is not set."
+model = os.getenv("LLM_MODEL", "openhands/claude-sonnet-4-5-20250929")
+base_url = os.getenv("LLM_BASE_URL")
+llm = LLM(
+    model=model,
+    api_key=SecretStr(api_key),
+    base_url=base_url,
+    usage_id="agent",
+)
+
+cwd = os.getcwd()
+
+
+# Initialize main agent with delegation capabilities
+main_agent = get_default_agent(llm=llm, enable_delegation=True, cli_mode=True)
```

Comment:
Shall we remove the `enable_delegation=True,` from the default agent, and manually add it to the tool set in this example to make it more explicit